go/exorelay/features/outgoing-replies.feature

```diff
@@ -0,0 +1,44 @@
+Feature: Sending outgoing replies to incoming messages
+
+  As an Exosphere developer
+  I want my code to be able to send replies to incoming Exosphere messages
+  So that I can write responsive ExoServices.
+
+  Rules:
+  - call the "reply" argument given to your message handler
+    to send a reply to the message you are currently processing
+
+
+  Background:
+    Given an ExoRelay with the role "test-service"
+    And ExoRelay connects to Exocom
+
+
+  Scenario: sending a reply
+    Given I register a handler for "users.create" which replies with "users.created" and the payload:
```

Comment:
This example is lacking the relevant parts, i.e. how the API is to send a reply looks like. They are buried into the step definition. It would be nice to see this more directly somehow. In a compiled language that's not really possible, unfortunately.

We could create an `examples/send-reply.go` file that defines the handler. Then load it here:

```cucumber
Given I register the "examples/send-reply" handler for "users.create" 
When receiving this message: ...
```

This way, it is at least possible to see how the handler looks like.